swift/mt940_test.go

```diff
@@ -153,66 +153,94 @@ func TestTransactionTagUnmarshal(t *testing.T) {
 }
 
 func TestBookingDateBug(t *testing.T) {
+
 	tests := []struct {
-		testdata            string
-		expectedBookingDate string
+		today                           string
+		balanceStartBookingDateString   string
```

Comment:
As we are already using the dates as `time.Time` in line 202ff we should change the type from `string` to `tim.Time`. Also, the setup gets easier as we don't have to parse the date strings